src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  $cardWidth: 200px;
+  display: grid;
+  grid-template-columns: repeat(4, $cardWidth);
+  gap: 48px;
+  justify-content: center;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 768px) and (max-width: 1024px) {
```

Comment:
Use one condition at a time by putting media queries in the correct order.
Example:
default -> media (max-width: 1023px) -> media (max-width: 767px) -> media (max-width: 477px)

Reply:
Media queries revised